andela/bp-esa-frontend
# #164044268 Fix filtering by date not returning reports for end date

#### What does this PR do?
Make filtering by date return reports for the end date

#### Description of Task to be completed?
- add 24 hours (84600000 ms) to the end date to account for time defaulting to 12 am

#### How should this be manually tested?
1. On the terminal, run: `yarn test` to observe that all tests pass
2. Launch the application and go to the browser to confirm that filtering by date now returns reports created on the `end date`

#### What are the relevant pivotal tracker stories?
[#164044268](https://www.pivotaltracker.com/story/show/164044268)
